amundsen_common/models/table.py

```diff
@@ -111,6 +113,8 @@ class Table:
     cluster: str
     schema: str
     name: str
+    # NB: This value is required for serialization in GET APIs, but ignored for PUT/POSTs
+    key: Optional[str] = None
```

Comment:
is this key same as table uri?

Reply:
Precisely! There's some frontend code that re-generates this.